website/docs/client/python.md

```diff
@@ -951,3 +950,137 @@ for event in events:
 The resulting lineage events received by Marquez would look like this.
 
 ![the Marquez graph](./mqz_graph_example.png)
+
+
+##### User-supplied Tags with Environment Variables 
+
+Integrations can add [tag facets](https://github.com/OpenLineage/OpenLineage/blob/main/proposals/3169/tags_facet.md) to runs, jobs and datasets. To allow more control over tags, users can add to and override integration-supplied tags through environment variables supplied to the client. The following rules apply to user-supplied tags. 
+
+* Tag environment variables follow this format. OPENLINEAGE__TAGS__JOB/RUN__KEY="VALUE"
```

Comment:
It's explained somewhere else so I guess it should be suggested to use it in a way described here.
FYI you can also set:
`OPENLINEAGE__TAGS='{"job":{"key":"value"},"run":{"key":"value"}}'` as well. The benefit it is you can do it in single env var.